frontend/test-utils.tsx

```diff
@@ -62,7 +74,9 @@ export function renderWithProviders(
           <AuthProvider>
             <QueryClientProvider client={new QueryClient()}>
               <I18nextProvider i18n={i18n}>
-                {children}
+                <ConversationProvider>
+                  {children}
+                </ConversationProvider>
```

Comment:
It will be safer to wrap the conversation provider around/before `QueryClientProvider` so that we are sure the requests we define with `useQuery` and `useMutation` have access to the `useConversation` hook

Reply:
Done!